buku.1

```diff
@@ -25,6 +25,7 @@ URLs are unique in DB. The same URL cannot be added twice. You can update tags a
 \fIDelete\fR operation:
   - When a record is deleted, the last record is moved to the index.
   - Delete doesn't work with range and indices provided together as arguments. It's an intentional decision to avoid extra sorting, in-range checks and to keep the auto-DB compaction functionality intact. On the same lines, indices are deleted in descending order.
+  - Can delete the search results, when any of the search flag is combined with -d.
```

Comment:
Can delete bookmarks matching a search, when any of the search flags is combined with -d.